src/daemon/daemon.cpp

```diff
@@ -1910,6 +1898,28 @@ void mp::Daemon::start_mount(const VirtualMachine::UPtr& vm, const std::string&
                      Qt::QueuedConnection);
 }
 
+std::string mp::Daemon::check_instance_alive(const std::string& instance_name) const
```

Comment:
IMO, the name of this method implies that we are checking that the instance is up and running.  I think maybe a better name would be something like `check_instance_usable` or `check_instance_startable`.

Reply:
You're right, I was struggling with naming there... From your suggestions I prefer `usable`. I am still not sure it conveys the meaning we want (not deleted), but it's certainly usable :stuck_out_tongue: 

Ahh I am sure there is an English word for it, I just can't seem to find it...

Reply:
Lets go with `operational`:)

Reply:
OK, sounds good, will do that.